custom_code/templatetags/target_list_extras.py

```diff
@@ -13,3 +17,20 @@ def target_list_extra_field(target_list, name):
         return TargetListExtra.objects.get(target_list=target_list, key=name).value
     except TargetListExtra.DoesNotExist:
         return None
+
+@register.inclusion_tag('tom_targets/partials/galaxy_table.html')
+def galaxy_table(target):
+    """
+    Displays the most likely host galaxy matches.
+    """
+    extras = {k['name']: target.extra_fields.get(k['name'], '') for k in settings.EXTRA_FIELDS if not k.get('hidden')}
+    return {
+        'target': target,
+        'extras': extras
+    }
+
+@register.filter
+def get_host_galaxies(target):
+    te = TargetExtra.objects.get(target=target, key='Host Galaxies')
```

Comment:
Can you first check if `te` is empty and return something reasonable? Right now the page crashes if there are no host galaxies.